### ISSUE TITLE
using an empty namespace in a different assembly does not produce a helpfull error message

### ISSUE THREAD

**Issue by pdelvo** [dotnet/roslyn#3063]
Consider this code:

``` csharp
using Assembly2;

public class Class1
{
    static unsafe void Main()
    {
    }
}
```

``` csharp
namespace Assembly2
{
}
```

Both files have to be in different projects. When you try to compile it in VS or with msbuild, csc will just return error code 1. The problem is the using Assembly2 in the first file. The namespace Assembly2 is empty.

**Comment by gafter** [comment]
@MadsTorgersen What does the language spec say about the *existence* of `Assembly2`? The compilers currently consider an imported namespace to exist only when it is nonempty.

**Comment by gafter** [comment]
Issue moved to [dotnet/csharplang #498](https://github.com/dotnet/csharplang/issues/498) via [**ZenHub**](https://www.zenhub.com/)
